Practice3/Program.cs

```diff
@@ -1,15 +1,62 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Practice3
 {
     class Program
     {
-        static void Main(string[] args)
+        private static int LowerBorder { get; set; }
+        private static int UpperBorder { get; set; }
+        private static int SumOfNumbersDevidableOn3AndNotDevidableOn5 { get; set; }
+        public static void Main(string[] args)
         {
+            Console.WriteLine("Set the lower value of the range!");
+            LowerBorder = Validator.GetValidatedInput();
+
+            Console.WriteLine("\nSet the upper value of the range!");
+            UpperBorder = Validator.GetValidatedInput();
+
+            if (IsRangeGoodToCreateArray(LowerBorder, UpperBorder))
+            {
+                int[] newArray = (int[])GetArrayOfNaturalNumbers(LowerBorder, UpperBorder);
+                GetSumOfNumbersDevidableOn3AndNotDevidableOn5(newArray);
+            }
+            else
+            {
+                Console.WriteLine("Less then 11 values range, exit");
+            }
+             Console.ReadKey();
+        }
+
+        private static bool IsRangeGoodToCreateArray(int lowerBorder, int upperBorder)
+        {
+            return ((UpperBorder - LowerBorder + 1) > 10);
+        }
+
+        private static Array GetArrayOfNaturalNumbers(int lowerBorder, int upperBorder)
+        {
+            Console.WriteLine("\nArray is below");
+            int[] numbersArray = new int[UpperBorder - LowerBorder + 1];
+            int i = 0;
+            do
+            {
+               numbersArray[i] = LowerBorder + i;
+               Console.WriteLine("Array element with i={0} is {1}", i, numbersArray[i]);
+               i++;
+            }
+            while ((LowerBorder + i <= UpperBorder) && (i <= UpperBorder - LowerBorder));
+            return numbersArray;
+        }
+
+        private static void GetSumOfNumbersDevidableOn3AndNotDevidableOn5(int [] array)
+        {
+            for (int i = 0; i < UpperBorder - LowerBorder; i++)
```

Comment:
It makes more sense to use array length instead of the "UpperBorder - LowerBorder"